src/App.jsx

```diff
@@ -33,6 +33,7 @@ function App() {
             <Nav />
             <Routes>
               <Route path="/" element={<ArticleList />} />
+              <Route path="/t/:topicName" element={<ArticleList />} />
```

Comment:
I'm assuming that "t" is short for topic here, but it might be nicer for the user to be able to see "topic" in the URL for clarity